### PR TITLE
chore(dependencies): update all our production dependencies

### PR DESCRIPTION
**Pull request by seanmonstar**

### CODE REVIEW

**Comment by seanmonstar** [comment]
Hm, oh joy. No git in docker image, can't depend on eslint-plugin-fxa git repo...

File: package.json [hunk 1]
```diff
@@ -29,19 +29,18 @@
   "license": "MPL-2.0",
   "dependencies": {
     "base64url": "2.0.0",
-    "bluebird": "2.1.3",
-    "clone": "1.0.2",
-    "convict": "1.4.0",
+    "bluebird": "3.5.0",
```
Comment (vladikoff) [inline package.json]: this might break things

**Comment by vladikoff** [comment]
could we split dep updates from eslint stuff?

**Comment by seanmonstar** [comment]
Yea, considered part of the update since I was upgrading from eslint-config to eslint-plugin... grumble.

**Comment by vladikoff** [comment]
i guess we need to add git to docker or something

File: package.json (hunk 1 above)
Comment (seanmonstar) [inline package.json]: I've made the changes for the upgrade to bluebird in this commit: change `done` to `then`, and update a `promisfy` call.

**Comment by seanmonstar** [comment]
Interesting, telling circle to rebuild without cache fixed it. All green!

I'll leave eslint for another PR.

**Comment by vladikoff** [comment]
Thanks Sean!

**Pull request merged**